web/styles/app_components.css

```diff
@@ -366,6 +366,11 @@ div.overlay {
     border-radius: 5px;
     box-shadow: none;
     margin: 0;
+
+    &.invalid {
```

Comment:
I think would be better to add this CSS to the specific element by using `#edit_user_full_name` selector for now since we only use this for one input currently in `settings.css` which contains other CSS for this UI.